README.md

```diff
@@ -252,6 +252,10 @@ For fluenbit,input, filter, and output plugins are connected by label selectors.
 
 However, for filter plugins, if you want a filter chain, the order of filters matters. You need to organize multiple filters into an array as the demo [logging stack](manifests/logging-stack/filter-kubernetes.yaml) suggests.For more info on various use cases of Fluent Operator Fluentd CRDs, you can refer to [Fluent-Operator-Walkthrough](https://github.com/kubesphere-sigs/fluent-operator-walkthrough#fluent-bit--fluentd-mode).
 
+### User Guides
```

Comment:
Shouldn't be `user guides`
Should be `Misc`

Reply:
fixed